package.json

```diff
@@ -1,96 +1,79 @@
 {
-  "name": "@nilfoundation/niljs",
-  "author": "=nil; Foundation",
-  "version": "0.13.0",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/NilFoundation/nil.js.git"
-  },
-  "homepage": "https://github.com/NilFoundation/nil.js.git#README.md",
-  "bugs": "https://github.com/NilFoundation/nil.js/issues",
-  "engines": {
-    "node": ">=18.0.0"
-  },
-  "type": "module",
-  "main": "dist/niljs.cjs",
-  "module": "dist/niljs.esm.js",
-  "files": [
-    "dist"
-  ],
-  "types": "dist/niljs.d.ts",
-  "exports": {
-    ".": {
-      "types": "./dist/niljs.d.ts",
-      "import": "./dist/niljs.esm.js",
-      "require": "./dist/niljs.cjs"
-    }
-  },
-  "sideEffects": false,
-  "description": "Typescript library to interact with the Nil blockchain. Can be used in the browser or in Node.js.",
-  "scripts": {
-    "test:unit": "vitest -c ./test/vitest.config.ts",
+	"name": "@nilfoundation/niljs",
```

Comment:
I would prefer not to format package.json file.
Seems that we have different formatter settings, this will result in constant updates of the file whenever somebody changes even a line of code.
We better sync formatter settings.

Reply:
turned off JSON formatting on my end for now